d2bs/kolbot/libs/OOG.js

```diff
@@ -975,15 +1028,31 @@ MainLoop:
 					do {
 						text = control.getText();
 
-						if (text instanceof Array && typeof text[1] === "string" && text[1].toLowerCase() === info.charName.toLowerCase()) {
-							control.click();
-							this.click(6, 627, 572, 128, 35);
+						if (text instanceof Array && typeof text[1] === "string") {
+							count++;
+
+							if (text[1].toLowerCase() === info.charName.toLowerCase()) {
+								control.click();
+								this.click(6, 627, 572, 128, 35);
 
-							break;
+								break MainLoop;
+							}
 						}
 					} while (control.getNext());
 				}
 
+				if (count === 8 || count === 16) { // check for additional characters up to 24
+					control = getControl(4, 237, 457, 72, 93);
+
```

Comment:
Do we need mouseblock here?

Reply:
Yes, we don't want people moving mouse around to click somewhere else

Reply:
This section is missing a mouse block

Reply:
The function already has mouse block set in the beginning.